ch04.adoc

```diff
@@ -197,28 +197,20 @@ The `computed_standard_name` attribute indicates that the values in variable
-----
-
-The specification:
-
-    seconds since 1992-10-8 15:15:42.5 -6:00
+Variables representing time must always explicitly include the **`units`** attribute; there is no default value. The **`units`** attribute takes a string value formatted as per the recommendations in the <<UDUNITS>> package. The following excerpt from the UDUNITS documentation explains the time unit encoding by example:
 
+"The specification `seconds since 1992-10-8 15:15:42.5 -6:00`
 indicates seconds since October 8th, 1992  at  3  hours,  15
 minutes  and  42.5 seconds in the afternoon in the time zone
 which is six hours to the west of Coordinated Universal Time
 (i.e.  Mountain Daylight Time).  The time zone specification
-can also be written without a colon using one or  two-digits
+can also be written without a colon using one or two digits
 (indicating hours) or three or four digits (indicating hours
-and minutes).
-      
-----
-
- 
+and minutes)."
 
-The acceptable units for time are listed in the link:$$http://www.unidata.ucar.edu/software/udunits/$$[`udunits.dat`] file. The most commonly used of these strings (and their abbreviations) includes **`day (d)`**, **`hour (hr, h)`**, **`minute (min)`** and **`second (sec, s)`**. Plural forms are also acceptable. The reference time string (appearing after the identifier **`since`**) may include date alone; date and time; or date, time, and time zone. The reference time is required. A reference time in year 0 has a special meaning (see <<climatological-statistics>>).
+The acceptable units for time are listed in the UDUNITS database. The most commonly used of these strings (and their abbreviations) includes **`day`** (**`d`**), **`hour`** (**`hr`**, **`h`**), **`minute`** (**`min`**) and **`second`** (**`sec`**, **`s`**). Plural forms are also acceptable.
 
-__Note: if the time zone is omitted the default is UTC, and if both time and time zone are omitted the default is 00:00:00 UTC.__ 
+The reference date/time string (appearing after the identifier **`since`**) may include date alone, or date and time, or date, time and time zone. The reference date/time string is required. If the time zone is omitted the default is UTC, and if both time and time zone are omitted the default is 00:00:00 UTC.
 
 We recommend that the unit **`year`** be used with caution. The Udunits package defines a **`year`** to be exactly 365.242198781 days (the interval between 2 successive passages of the sun through vernal equinox). __It is not a calendar year.__ Udunits includes the following definitions for years: a **`common_year`** is 365 days, a **`leap_year`** is 366 days, a **`Julian_year`** is 365.25 days, and a **`Gregorian_year`** is 365.2425 days.
 
```

Comment:
Perhaps this is a good opportunity to update the second `udunits.dat` link in the following line 217 as well.

I also wonder if we should be explicit in whether CF follows udunits in the definitions of year and month. The way these two paragraphs are written, the reader might come away with the impression that we are only warning of potential misinterpretation by some other software, whereas my understanding so far is that we adopt the udunits definition, albeit begrudgingly. But perhaps my understand is wrong or you would prefer to address this in a separate issue?